examples/app_tdm_loopback_demo/Makefile

```diff
@@ -18,8 +18,8 @@ USED_MODULES = lib_gpio(>=1.1.0) lib_i2c(>=5.0.0) lib_i2s(>=3.0.0)
 # xcc for the final link (mapping) stage.
 XCC_FLAGS = -O3 -report -g
 
-test: bin/i2s_loopback_demo.xe
-	xsim bin/i2s_loopback_demo.xe --plugin LoopbackPort.dll '-port tile[0] XS1_PORT_1D 1 0 -port tile[1] XS1_PORT_1E 1 0'
+# test: bin/i2s_loopback_demo.xe
```

Comment:
Did you comment out these demos as a permenant or temporary step?

Reply:
fixed in latest commit